entries/t/tryhackme.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Try Hack Me": {
```

```suggestion
  "TryHackMe": {
```

The name doesn't have any spaces in between on the site.